models/a-2084-01/a-2084-01.sdf

```diff
@@ -1,6 +1,6 @@
 <sdf version='1.6'>
-  <model name='A-2084-01'>
-    <link name='HEBI/Base/INPUT_INTERFACE'>
+  <model name='a-2084-01'>
+    <link name='HEBI/base/INPUT_INTERFACE'>
```

Comment:
I'm not against these being lowercase, but we should make sure we're consistent (ideally with other APIs, too).  Out of curiosity, what was the motivation for this?  I assume this matches usage somewhere better.

Reply:
I probably was trying to test code on a robot that had a different actuator naming convention, and just regenerated the models to match. These changes aren't really intended to be "official" because I regenerate the sdf models all the time after making xacro/hrdf changes. Ideally we wouldn't commit anything to the repo except xacro and hrdf, but I didn't want to require users to run the conversion pipeline, especially if they aren't using ROS and don't have xacro installed. The case of these paths is set by whatever is in https://github.com/HebiRobotics/hebi_description/blob/master/tools/generate_pipeline_batch_v1.bash, so it can be changed to whatever makes sense. Maybe actuator names should be saved in a yaml file in the package instead of being baked into the bash script?

Reply:
I have been thinking about this more. Maybe it would be better to avoid baking the actuator and family names into the urdf until runtime? So instead of doing `xacro --xacro-ns hebi_arm.xacro` which is what we do now, it'd be `xacro --xacro-ns hebi_arm.xacro family:=HEBI actuators:="A1 A2 A3 ..."`, and they aren't resolved until the time the xacro is loaded into robot_description

Pro:
* User must explicitly set actuator and family names when loading xacro - less likely to forget to update with robot changes.
* I Can make a modified Moveit! config which pulls from that same yaml at run time, saving users from having to regenerate Moveit! configs when they change actuator/family name
    * This presumes I can figure out a way to template SRDF files. From a quick inspection this should not be hard, though it depends on how Moveit! loads the file.
* We should be able to compose kit xacros (eg 6DOF arm and Rosie) without running into issues with hard coded actuator families.

Con:
* I'm not sure how much work maintaining a non-standard Moveit! config will create down the road.
* Deviates a bit from the normal workflow of loading a model into a launchfile
* Actual syntax might be a good deal messier than this. Specifically, I don't think passing lists into an xacro arg is as pleasant as my hypothetical example here. Need to do more reading though.
* This doesn't benefit any non-ROS users, as they would be using the static SDF files.

I'm going to sit on this for a few days while I work on some other stuff, since the current code is (basically) functional and I need to tackle some other PRs. Let me know if you have any thoughts.